code/src/component/Item.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducer/todos'
+import moment from 'moment'
+import styled from 'styled-components'
+
+
+const ItemContainer = styled.div`
+    margin-bottom: 15px;
+`
+const TaskContainer = styled.li`
+    width: 100%;
+    height: 18px;
+    display: flex;
+    align-items: center;
+    padding: 26px;
+    margin-bottom: 5px;
+    font-size: 16px;
+    color: #3f3f3f;
+    background-color: ${props => props.checked ? "#F0F0F0" : "#E1C9FD"};
+    text-decoration: ${props => props.checked ? "line-through" : "none"};
+    border-radius: 45px;
+    list-style-type: none;
+    font-family: 'Montserrat', sans-serif;
+    box-sizing: border-box;
+    justify-content: space-between;
+`
```

Comment:
Great usage of styled components! Looks really good when you check a task and the line through is displayed 🎖️